sub-projects/hub/src/server/aws/aws_s3.js

```diff
@@ -0,0 +1,42 @@
+const AWS = require('aws-sdk');
```

Comment:
The file naming convention for this project uses dashes `-`, not underscores `_`.

Reply:
Also, this should probably go in either the `personofinterest` directory or the `utils` directory, since it's not an endpoint.